### DISCUSSION TITLE
Change the way to assign issue

### DISCUSSION THREAD

**Discussion by UG-SEP** [MakeContributions/DSA#342] [Ideas]
Hey @ming-tsai , @Arsenic-ATG, @atin 
I am thinking that we should automate to assign issue work `i.e.`: Open an issue and if you want to handle or get assign then write 
`#assignme` and the issue will assign to you...
## The reason behind it:
 It takes a lot of time for the developer to get assign that issue first to open an issue and then wait till the issue does not assign to them by the maintainer. So it takes a lot of the developer 
`What you thought about the idea please comment below`

**Comment by Arsenic-ATG** [comment]
What if another person comes in and creates a duplicate issue with `#assignme` tag ? wouldn't it create redundancy and destroy the purpose of getting assigned and reserving that task for them self at the first place ?

**Reply by UG-SEP** [reply]
you are right it is not too necessary because there are not too much issue open at a particular time and well you are right
>What if another person comes in and creates a duplicate issue with the #assignme tag? wouldn't it create redundancy and destroy the purpose of getting assigned and reserving that task for themself in the first place ?
It fails because it works is only to assign. It's doesn't check whether it is duplicate or not 
Well @Arsenic-ATG  Do you know how to create this type of bots for github repos

**Comment by lubnafathima** [comment]
> Hey @ming-tsai , @Arsenic-ATG, @atin I am thinking that we should automate to assign issue work `i.e.`: Open an issue and if you want to handle or get assign then write `#assignme` and the issue will assign to you...
> 
> ## The reason behind it:
> It takes a lot of time for the developer to get assign that issue first to open an issue and then wait till the issue does not assign to them by the maintainer. So it takes a lot of the developer `What you thought about the idea please comment below`

hey, so the issue was accepted?

**Reply by ming-tsai** [reply]
I guess we accepted your comment, it is going to assign manually for now.
